packages/react-charting/src/utilities/ChartTitle.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { hasOverflow, ITooltipHostProps, TooltipHost, TooltipOverflowMode } from '@fluentui/react';
+import { getAccessibleDataObject } from './index';
+import { IAccessibilityProps } from '../types/index';
+
+interface IChartTitleProps {
+  className?: ITooltipHostProps['hostClassName'];
+  accessibilityData?: IAccessibilityProps;
+  children?: ITooltipHostProps['content'];
+}
+
+interface IChartTitleState {
+  textOverflow: boolean;
+}
+
+export class ChartTitle extends React.Component<IChartTitleProps, IChartTitleState> {
```

Comment:
this component needs to be renamed. This is not chart title

Reply:
renamed it to HTMLTooltipText. Any suggestions?